pyproject.toml

```diff
@@ -25,7 +25,7 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.8"
```

Comment:
remove classifiers for 3.7
classifiers = [
    .
    .
    "Programming Language :: Python :: 3.7",
]

Reply:
Updated